### DISCUSSION TITLE
Better navigation in feed set-up wizard

### DISCUSSION THREAD

**Discussion by piotrpog** [engram-design/FeedMe#1012] [Ideas]
Right now if we are in step 3 and want to change something in step 1, we need to go through step 2. 

It would be better if we could go to step 1 directly in such a situation.

**Comment by kristiansp** [comment]
@piotrpog If you navigate via the breadcrumbs in the top, you will be able to jump from step 3 straight back to step 1.

**Comment by piotrpog** [comment]
@kristiansp But not from step 1 to step 3. Assuming that we are editing already existing feed, we might want that.

**Comment by kristiansp** [comment]
@piotrpog Yeah, this is true. It has annoyed me too. It's not a great solution, but if you go directly to `/admin/feed-me/feeds/map/<feedId>` it will work. But you would have to do it manually or have a bookmark.

I guess the reason is that if you change something on the first two pages, the third could be rendered useless, so it's made as one flow, but it's extremely cumbersome when wanting to edit just the mapping.

I was a bit confused as the original post referred to moving from page 3 to 1. But going straight to the mapping for an existing feed would be a really useful change.
